linters/check_columns.py

```diff
@@ -456,7 +465,16 @@ def check_profile():
                 print_options(result)
                 delete_command(result, "profile_columns.list")
 
+    return len(result)
+
 
 if __name__ == "__main__":
-    check_history()
-    check_profile()
+    result1 = check_history()
+    result2 = check_profile()
+
+    result = result1 or result2
+    if result == 0:
```

Comment:
I believe this is more "Pythonic":
```suggestion
    if not result:
```